packages/admin/admin/src/common/Tooltip.tsx

```diff
@@ -236,7 +239,7 @@ export const Tooltip = (inProps: TooltipProps) => {
                 disableTouchListener
                 {...commonTooltipProps}
             >
-                {cloneElement(children, { onClick: toggleTooltip })}
+                {cloneElement(children as ReactElement<any, any>, { onClick: toggleTooltip })}
```

Comment:
Why is this necessary?

Reply:
The type of `children` in MUIs `Tooltip` was changed from `ReactElement<any, any>` to `React.ReactElement<unknown, any>`. 

Since this change typescript no longer ignores that `onClick` might not be available on whatever element `children` is, so it is not allowed to set `onClick`. 

I wasn't exactly sure how to check if the element is supported and since this doesn't change anything in the behavior it didn't seem worth investing a lot of time into fixing this 😅